ui/pages/settings/flask/view-snap/index.scss

```diff
@@ -24,6 +24,19 @@
     }
   }
 
+  &__install-details {
+    border-bottom: 1px solid var(--color-border-muted);
+    margin-right: 24px;
```

Comment:
We might want to try `margin-inline-end` for RTL support.

Reply:
Done 👍